src/lib.rs

```diff
@@ -127,9 +177,24 @@ impl Collector {
     /// # use metrics_process::Collector;
     /// let collector = Collector::default().prefix("my_prefix_");
     /// ```
-    pub fn prefix(mut self, prefix: impl Into<String>) -> Self {
-        self.prefix = prefix.into();
-        self
+    pub fn prefix(self, prefix: impl Into<String>) -> Self {
+        drop(self);
```

Comment:
This is just a question, but I thought `self` is automatically dropped once the function is called. Why do we need an explicit drop here?

Reply:
To be explicit :wink: And also to suppress the warning about unused `self`. Can be changed to something like `let _ = self;`

Reply:
Well, I prefer let _ = self; because it's clear that the line does nothing 👍